### PR TITLE
[Keyboard] Add eniigmakeyboards/ek60

### PR DESCRIPTION
**Pull request by eniigmakeyboards**
## Description

Added support for the ek60 PCB, designed for use with 60% Eniigma Keyboards products.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by eniigmakeyboards** [comment]
Travis CI build failed due to:
`docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.`

**Comment by eniigmakeyboards** [comment]
Just wanted to follow up on this PR as it has been marked for review.

File: keyboards/eniigmakeyboards/ek60/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/eniigmakeyboards/ek60/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/eniigmakeyboards/ek60/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/eniigmakeyboards/ek60/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/eniigmakeyboards/ek60/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (drashna) [inline keyboards/eniigmakeyboards/ek60/rules.mk:22]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = 60_ansi
```

And if any other layouts are community supported, they should be added here.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/eniigmakeyboards/ek60/keymaps/default/keymap.c (hunk 1 above)
Comment (eniigmakeyboards) [reply keyboards/eniigmakeyboards/ek60/keymaps/default/keymap.c:17]: Done.

File: keyboards/eniigmakeyboards/ek60/readme.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Eniigma Keyboards ek60
+
+* Keyboard Maintainer: [eniigmakeyboards](https://github.com/eniigmakeyboards) and [adamws](https://github.com/adamws)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/eniigmakeyboards/ek60/readme.md:5]: Could you add instructions on how to reset to the bootloader, to flash new firmware?

Comment (eniigmakeyboards) [reply keyboards/eniigmakeyboards/ek60/readme.md:5]: Done

**Review by drashna — APPROVED** [review]

**Comment by eniigmakeyboards** [comment]
Can this please be looked over/approved soon? It's been nearly 2 months since the PR was opened. Thank you.

File: keyboards/eniigmakeyboards/ek60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+LAYOUTS = 60_ansi, 60_iso
```
Comment (fauxpark) [inline keyboards/eniigmakeyboards/ek60/rules.mk:24]:
```suggestion
LAYOUTS = 60_ansi 60_iso
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by eniigmakeyboards** [comment]
Sorry to continue bumping this - but it'd be much appreciated if we can wrap this up soon! It's been open since early June, and would like to get it done.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**